databuilder/extractor/dashboard/tableau/tableau_dashboard_extractor.py

```diff
@@ -39,6 +40,10 @@ def execute(self) -> Iterator[Dict[str, Any]]:
                           if workbook['projectName'] not in
                           self._conf.get_list(TableauGraphQLApiMetadataExtractor.EXCLUDED_PROJECTS, [])]
         base_url = self._conf.get(TableauGraphQLApiMetadataExtractor.TABLEAU_BASE_URL)
+        site_name = self._conf.get_string(TableauGraphQLApiMetadataExtractor.SITE_NAME)
+        site_url_path = ""
+        if site_name != "":
```

Comment:
Please use single quotes (`''`) here for consistency.

Reply:
fixed